tools/json2bin/meson.build

```diff
@@ -2,3 +2,4 @@ encounters_py = find_program('encounters.py', native: true)
 movedata_py = find_program('movedata.py', native: true)
 pokemon_personal_data_py = find_program('pokemon_personal_data.py', native: true)
 pokemon_wotbl_data_py = find_program('pokemon_wotbl_data.py', native: true)
+evo_py = find_program('evo.py', native: true)
```

Comment:
Make sure this file ends with an empty new-line.

Reply:
On my end it looks like I have the empty new-line at the end, but it's not showing up on git.

Reply:
Missing new-lines show up in github diffs as a ⛔ symbol at the end of a file.